articrap.lua

```diff
@@ -0,0 +1,48 @@
+--A half-assed way to get artifact quality gear without the actual artifact or artifake.
```

Comment:
Not a great script name

Reply:
Hmmm, artifake for this and artimake for the artifake script might make more sense?